client/create.go

```diff
@@ -24,16 +34,25 @@ For more about knox, see https://github.com/pinterest/knox.
 See also: knox add, knox get
 	`,
 }
+var createTinkKeyset = cmdCreate.Flag.Bool("key-template", false, "")
 
 func runCreate(cmd *Command, args []string) {
-	if len(args) != 1 {
-		fatalf("create takes exactly one argument. See 'knox help create'")
+	if len(args) != 1 && len(args) != 2 || len(args) == 2 && !*createTinkKeyset ||
+		len(args) != 2 && *createTinkKeyset {
+		fatalf("unsupported command. See 'knox help create'")
 	}
-	fmt.Println("Reading from stdin...")
-	keyID := args[0]
-	data, err := ioutil.ReadAll(os.Stdin)
-	if err != nil {
-		fatalf("Problem reading key data: %s", err.Error())
+	var keyID string
+	var data []byte
+	if *createTinkKeyset {
+		templateName := args[0]
+		keyID = args[1]
+		if err := checkTemplateNameAndKnoxIDForTinkKeyset(templateName, keyID); err != nil {
```

Comment:
it's ok to split this onto two lines

Reply:
Thank you so much! I have revised my code according to your comment.